apps/meteor/client/views/admin/manageSubscription/components/FeatureUsageCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box } from '@rocket.chat/fuselage';
+import { Card, CardBody, CardColSection, CardFooter, CardTitle } from '@rocket.chat/ui-client';
+import type { ReactElement, ReactNode } from 'react';
+import React, { memo } from 'react';
+
+import InfoTextIconModal from './InfoTextIconModal';
+import UpgradeButton from './UpgradeButton';
+
+export type FeatureUsageCardProps = {
+	title: string;
+	children?: ReactNode;
+	infoText?: string;
+	showUpgradeButton?: boolean;
```

Comment:
you used the props directly right 👍 
do the same on all cards instead of card object

Reply:
Is there a reason for that? Personally I find more organized and easy to read

Reply:
Actually I changed this component to use the card object instead, following the recommendation at: https://alexkondov.com/tao-of-react/#objects-instead-of-primitives

Reply:
Sounds good to me!